crates/file/src/blob.rs

```diff
@@ -189,7 +197,14 @@ impl File {
     where
         T: BlobContents,
     {
-        Self::new_(name, contents.into_jsvalue(), mime_type, last_modified_time)
+        Self::new_(
+            name,
+            // SAFETY: `Self::new_` doesn't modify the `Uint8Array`,
+            // or hold on to the reference past the end of the function call.
+            unsafe { contents.into_jsvalue() },
```

Comment:
I think we should pass `contents` to `Self::new_`. Currently, someone could change `Self::new_` without realizing it this safety requirements. Also, that would reduce the number of places `unsafe` block is required.